apps/web-giddh/src/app/shared/header/header.component.html

```diff
@@ -365,8 +365,8 @@ <h6 class="mrB1">MENU
                                         </li>
                                         <li><a (click)="analyzeOtherMenus('contact/vendor', {tab: 'vendor', tabIndex: 0})">Vendor</a>
                                         </li>
-                                        <li><a (click)="analyzeOtherMenus('accounting-voucher')">Journal Voucher</a>
-                                        </li>
+                                        <!-- <li><a (click)="analyzeOtherMenus('accounting-voucher')">Journal Voucher</a>
+                                        </li> -->
```

Comment:
will this commented code reuse in future?

Reply:
It's JV code commented for some time as new JV module is under development.